docs/content/content/content-principles.mdx

```diff
@@ -119,13 +119,13 @@ Make sure there are clear exit points and navigation to previous or next steps i
 
 ## Unique
 
-While content should be consistent across the Export Service, it should not be uniform or duplicated.
+Content should be consistent across the Export Service. But that does not mean it should be uniform or duplicated.
 
 Avoid using significant chunks of the same content across pages. It’s a bad user experience and may lower page rankings in search engine results.
 
-Content for each web page should be unique so users and search engines can clearly understand the difference between them and easily select a relevant page.
+Content for each web page should be unique. Users and search engines can then clearly understand the difference between pages. It allows them to easily select a relevant page.
 
-If content already appears on the Export Service or the [Department of Agriculture, Fisheries and Forestry](https://www.agriculture.gov.au/) (DAFF) website, link to the authoritative source.
+If content already appears somewhere online link to the authoritative source. For example, the [Export Service](https://exports.agriculture.gov.au/). Or the [Department of Agriculture, Fisheries and Forestry](https://www.agriculture.gov.au/) (DAFF) website.
```

Comment:
Feels weird starting this sentence with "or", shouldn't it just be a continuation? For example, the Export Service of the DAFF website

Reply:
@Suzanne-aff As discussed